config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
   namespace :api do
+    get 'products/index'
+  end
+  namespace :api do
+    resources :products, only: %i[index show]
     namespace :v0 do
       resources :pings, only: [:index], constraints: { format: 'json' }
```

Comment:
same with this, what are pings?

Reply:
It was auto generated by Rails, but we have removed it now.